packages/components/src/DateTimePickers/pickers/DateTimePicker/DateTimePicker.test.js

```diff
@@ -227,4 +227,22 @@ describe('DateTimePicker', () => {
 			expect(onSubmit).toBeCalledWith(event, { date: initialDate, time });
 		});
 	});
+
+	describe('today function', () => {
```

Comment:
Can you add a test that check that the picked value is selected too ?
You can look at the other tests to have examples

Reply:
I have this 
`expect(wrapper.state('selectedDate')).toStrictEqual(startOfDay(new Date()));`

Reply:
There is 
`expect(wrapper.state('selectedDate')).toStrictEqual(startOfDay(new Date()));`